client/src/components/scheduled-events/Event.vue

```diff
@@ -185,6 +218,14 @@ export default {
                 .padStart(2, '0');
 
             return day + ' ' + month + ' ' + year;
+        },
+
+        async onCancelHandle() {
+            const updatedEvent = {
+                ...this.scheduledEvent,
```

Comment:
Why would you pass the entire state of the current event to change one field?

Reply:
I want use updateEvent action to the all update methods for event